packages/helm-charts/oracle/USDCEUR.yaml

```diff
@@ -28,7 +28,6 @@ oracle:
         {exchange: 'KRAKEN', symbol: 'EURUSD', toInvert: true}
       ],
       [
-        {exchange: 'BINANCEUS', symbol: 'USDCUSD', toInvert: false},
```

Comment:
Lines 30, 32 and 33 also have to be deleted

Reply:
ditto